python/across_api/base/common.py

```diff
@@ -16,6 +19,28 @@
 tzset()
 
 
+def round_time(t: Time, step: u.Quantity) -> Time:
+    """
+    Round an astropy Time object to a given time step. So for example, if
+    step=1*u.s, then the time will be rounded to the nearest second. If step
+    is 1*u.min, then the time will be rounded to the nearest minute.
+
+    Parameters
+    ----------
+    t
+        The time to round.
+    step
+        The time step to round to.
+
+    Returns
+    -------
+        The rounded Time.
+    """
+    return Time(
+        np.round(t.unix / step.to_value(u.s)) * step.to_value(u.s), format="unix"
+    )
```

Comment:
See https://docs.astropy.org/en/stable/time/#precision

Reply:
Precision is about decimal places when printing, right? This is about rounding the actual time value to whole seconds and whole minutes.